EasyPost/Services/CarrierAccountService.cs

```diff
@@ -37,19 +38,22 @@ internal CarrierAccountService(EasyPostClient client)
         /// <param name="cancellationToken"><see cref="CancellationToken"/> to use for the HTTP request.</param>
         /// <returns>A <see cref="CarrierAccount"/> object.</returns>
         [CrudOperations.Create]
+        [Obsolete("This function does not support FedEx or UPS account registrations. Please use the Create(Parameters.CarrierAccount.ACreate) function instead for all carrier account registrations.")]
```

Comment:
Question: has it been this way in the past? just want to confirm that we are not making breaking change

Reply:
In general, we're trying to drive users away from using the legacy dictionary-based methods to the new parameter-set-based methods. While this function could technically support custom workflows like FedEx (but not UPS), it's easier to just say that it's not recommended for either of them, and suggest users use the parameter-set alternatives instead to avoid any potential incompatibilities.